service_mesh/v1x/threescale-adapter.adoc

```diff
@@ -31,7 +31,7 @@ include::modules/ossm-threescale-metrics-1x.adoc[leveloffset=+1]
 include::modules/ossm-threescale-istio-adapter-verification.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-.Additional resources
+== Additional resources
```

Comment:
Given that this additional resources section is not at the end of the assembly and it seems specific to the preceding section, you could consider leaving this as `.Additional resources`.